packages/camel-catalog/kaoto-camel-catalog-maven-plugin/src/main/java/io/kaoto/camelcatalog/CamelCatalogProcessor.java

```diff
@@ -165,4 +169,21 @@ public String getPatternCatalog() throws Exception {
         });
         return JsonMapper.serialize(answer);
     }
+
+    private void populatePatterBlacklist() {
```

Comment:
```suggestion
    private void populatePatterBlocklist() {
```

Reply:
Thank you for bringing this up. I fixed that.